### PR TITLE
Use correct script while building volar

### PR DESCRIPTION
**Pull request by wvffle**
- [x] I have read the note above about PRs contributing or fixing extensions
- [ ] I have tried reaching out to the extension maintainers about publishing this extension to Open VSX (if not, please create an issue in the extension's repo using [this template](https://github.com/open-vsx/publish-extensions/blob/HEAD/docs/external_contribution_request.md)).
- [x] This extension has an [OSI-approved OSS license](https://opensource.org/licenses) (we don't accept proprietary extensions in this repository)

## Description

This PR applies @johnsoncodehk's suggestions from #626.

### CODE REVIEW

**Comment by filiptronicek** [comment]
Thanks a lot, @wvffle! 
Would you be able to look at the build errors with `Vue.vscode-typescript-vue-plugin` as well? If we fix something about the extension it would be nice to have it build successfully.

**Comment by wvffle** [comment]
@filiptronicek, I've tried to look at the errors. It is... weird... If I install the dependencies in the root directory with `npm`, there are typescript errors but when I install them with `pnpm` - it builds fine. But the `Vue.vscode-typescript-vue-plugin` plugin needs its own dependencies to be installed. If installed with `pnpm` - an `ELSPROBLEMS` error is thrown but when installed with `npm`, it works fine.

What I did feels very hacky but it seems to work and package the extension:
image

**Comment by johnsoncodehk** [comment]
@wvffle `Vue.vscode-typescript-vue-plugin` actually cannot pack via script, this is a limitation of `vsce`, I mush to do these steps manually:

1. run `pnpm install && npm run build` in repo root
2. remove `"dependencies": { "typescript-vue-plugin-forward": "file:typescript-vue-plugin-forward" },` in extensions/vscode-typescript-vue-plugin/package.json.
3. run `npm run pack` in extensions/vscode-typescript-vue-plugin.
4. add back `"dependencies": { "typescript-vue-plugin-forward": "file:typescript-vue-plugin-forward" },`

File: extensions.json [hunk 1]
```diff
@@ -521,16 +521,16 @@
...
-  "johnsoncodehk.vscode-typescript-vue-plugin": {
+  "Vue.vscode-typescript-vue-plugin": {
     "repository": "https://github.com/johnsoncodehk/volar",
     "location": "extensions/vscode-typescript-vue-plugin",
     "custom": [
-      "npx pnpm install --no-frozen-lockfile --shamefully-hoist",
+      "npx pnpm install",
       "npx pnpm run build",
-      "cd ./extensions/vscode-typescript-vue-plugin && npx vsce package -o extension.vsix"
+      "cd ./extensions/vscode-typescript-vue-plugin && npm install && npx vsce package -o extension.vsix"
```
Comment (johnsoncodehk) [inline extensions.json:533]: I think `npm install &&` is redundant here, you should install by pnpm in root dir.

**Comment by wvffle** [comment]
@johnsoncodehk thanks for explaining the build steps!

**Comment by filiptronicek** [comment]
Looking better now, @wvffle! 

Could you take a look at the `/bin/sh: 1: Syntax error: redirection unexpected` error? [[1](https://github.com/open-vsx/publish-extensions/runs/7641206343?check_suite_focus=true#step:7:246)]

**Comment by wvffle** [comment]
> Could you take a look at the `/bin/sh: 1: Syntax error: redirection unexpected` error? [[1](https://github.com/open-vsx/publish-extensions/runs/7641206343?check_suite_focus=true#step:7:246)]

Should work now

**Review by filiptronicek — APPROVED** [review]
Many thanks, @wvffle! 

LGTM ✅

**Comment by wvffle** [comment]
@filiptronicek, there seems to be an error while publishing volar to the repository: https://github.com/open-vsx/publish-extensions/runs/7693396759?check_suite_focus=true#step:9:6797, could you take a while and look at it? :)

**Comment by filiptronicek** [comment]
Thanks for noting this, @wvffle, I have taken a look at this and sadly it seems it's going to be blocked until open-vsx.org deploys next, which I'm not sure when it will be :/.

**Comment by anbraten** [comment]
@filiptronicek I think the `Vue` namespace is currently owned by @lukashass if that helps: https://github.com/johnsoncodehk/volar/pull/1255 
Could we help you somehow to unclaim it?

**Comment by filiptronicek** [comment]
@anbraten if I'm not mistaken, you cannot just get ownership of a namespace by creating it. You only are assigned as a member, you cannot add others to it therefore not an owner - to be an owner you need to claim the namespace, first. @open-vsx is kind of the BDFL of https://open-vsx.org, it can publish any extension at any time under any namespace. It shouldn't be the problem here. I will try to discuss with the team at Eclipse if they have any logs about the publishing errors.

**Pull request merged**